src/containers/course-sections-list/CourseSectionsList.styles.ts

```diff
@@ -20,5 +22,61 @@ export const styles = {
     transform: 'rotate(90deg)',
     color: 'primary.400',
     cursor: 'pointer'
+  },
+  activityButton: {
+    typography: TypographyVariantEnum.Body2,
+    color: 'basic.blueGray',
+    fontWeight: 500,
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    m: '0 7px'
```

Comment:
do you need to rewrite margin? if no, you can use mx

Reply:
Yep, actually i need to rewrite `my` too